cmd/draino/draino.go

```diff
@@ -142,33 +158,38 @@ func main() {
 		}
 	}
 
-	sf := cache.FilteringResourceEventHandler{FilterFunc: kubernetes.NodeSchedulableFilter, Handler: h}
```

Comment:
Trying to grok why we removed this:

```
commit 8bb413525663393c42665a92d636a5bd7da6ad1b
Author: Nic Cope <negz@planet.com>
Date:   Sun Aug 26 23:12:01 2018 -0700

    Filter out cordoned nodes

    We immediately cordon nodes, so this should stop us attempting to cordon and
    drain a node multiple times if we see multiple updates via our watch.
```

I'll continue reading but I presume we're relying on our new node conditions instead.

Reply:
Now we don't want to filter out cordoned nodes anymore. The reason is that if draino cordon a node, schedule a drain, and then the process terminate (without performing the drain), you want the new draino process to take into account this node again. The new draino process will see that the node is cordoned but that there is no schedule pending so it will create a new one:

https://github.com/planetlabs/draino/pull/67/files#diff-86d89adaf5925d6ff41a251e1317875cR145